client/src/components/Account/CustomTextField.tsx

```diff
@@ -0,0 +1,31 @@
+import { Grid, TextField, Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  formLabel: {
+    fontWeight: 700,
+  },
+}));
+
+interface Props {
+  label: string;
+  placeholder: string;
+}
+
+export default function CustomTextField({ label, placeholder }: Props): JSX.Element {
```

Comment:
Good thinking with modularizing this 👍 One thing you might want to add as a prop is passing a value and onChange function. When you need to integrate this in the future that will make getting the current values significantly easier

Reply:
Done!